packages/protocol/contracts/common/FeeHandlerSeller.sol

```diff
@@ -0,0 +1,86 @@
+pragma solidity ^0.5.13;
+
+import "openzeppelin-solidity/contracts/math/SafeMath.sol";
+import "../common/FixidityLib.sol";
+import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
+import "openzeppelin-solidity/contracts/token/ERC20/IERC20.sol";
+import "./UsingRegistry.sol";
+import "../common/Initializable.sol";
+
+contract FeeHandlerSeller is Ownable, Initializable, UsingRegistry {
+  using SafeMath for uint256;
+  using FixidityLib for FixidityLib.Fraction;
+
+  mapping(address => uint256) public minimumReports;
+
+  event MinimumReportsSet(address tokenAddress, uint256 minimumReports);
+  event TokenSold(address tokenAddress, uint256 amount);
```

Comment:
Can we extend this event to also log which token/amount it was exchanged for ?

Reply:
Shouldn't it always be Celo?